README.md

```diff
@@ -28,6 +28,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 <!-- toc -->
 
+- [Table of Contents](#table-of-contents)
```

Comment:
Do you add this manually?

Reply:
This is automatically generated by vscode formatting, I deleted it